app/Jobs/DeleteGenre.php

```diff
@@ -67,14 +63,14 @@ public function handle()
 
         // This is done instead of a single UPDATE query in order to
         // generate revision logs for the change.
-        $this->genreToDelete->tracks()->chunk(200, function ($tracks) {
```

Comment:
Removing this could vastly and fatally increase memory usage when deleting large genres. What did removing the chunking here fix?